src/routes/JobOpportunity/JobOpportunity.tsx

```diff
@@ -50,36 +49,157 @@ const JobOpportunity = () => {
   );
   const clusterSize = tyomahdollisuus?.jakaumat?.ammatti?.maara;
 
-  interface HeadingRoute {
-    title: string;
-    isDev?: boolean;
+  interface Section {
+    navTitle: string;
+    section: JSX.Element;
```

Comment:
Maybe it would be more readable if the `section` prop is named to something different than the interface, like `content` that describes it better 🤔

Reply:
Renamed, makes more sense now